### PR TITLE
unignore transRelationWithRelationGuardsAtBothEnds

### PR DESCRIPTION
**Pull request by kasper-piskorski**

### CODE REVIEW

**Comment by kasper-piskorski** [comment]
one travis pass

File: grakn-graql/src/test/java/ai/grakn/graql/internal/reasoner/ReasoningTests.java [hunk 1]
```diff
@@ -281,13 +288,21 @@ public void transRelationWithEntityGuardsAtBothEnds() {
...
 
-    @Ignore // TODO: Fix (Bug #16195)
     @Test //Expected result: The query should return a unique match
     public void transRelationWithRelationGuardsAtBothEnds() {
         QueryBuilder qb = testSet11.graph().graql().infer(true);
         String queryString = "match (role1:$x, role2:$y) isa relation3;";
-        QueryAnswers answers = queryAnswers(qb.parse(queryString));
-        assertEquals(answers.size(), 1);
+
+        System.out.println("rel1: ");
```
Comment (fppt) [inline grakn-graql/src/test/java/ai/grakn/graql/internal/reasoner/ReasoningTests.java]: Are all these `printlns` necessary? They going to make the logs on travis bigger.

Comment (kasper-piskorski) [inline grakn-graql/src/test/java/ai/grakn/graql/internal/reasoner/ReasoningTests.java]: No, they aren't. Felix said the test is failing randomly. It's just if the test fails I want to have an idea of what went wrong :)

Comment (fppt) [inline grakn-graql/src/test/java/ai/grakn/graql/internal/reasoner/ReasoningTests.java]: Alright. We will leave it as is then. But they should ideally be removed at some point.

Comment (fppt) [inline grakn-graql/src/test/java/ai/grakn/graql/internal/reasoner/ReasoningTests.java]: Oh wait. So the test is still failing randomly?

Comment (kasper-piskorski) [inline grakn-graql/src/test/java/ai/grakn/graql/internal/reasoner/ReasoningTests.java]: Dunno, Trying really hard but haven't managed to make it fail so far.

Comment (fppt) [inline grakn-graql/src/test/java/ai/grakn/graql/internal/reasoner/ReasoningTests.java]: With these annoying tests I usually go for like 10 retests on travis. Maybe try that? After ten retries you can poke me and I will merge. Just a suggestion though.

Comment (kasper-piskorski) [inline grakn-graql/src/test/java/ai/grakn/graql/internal/reasoner/ReasoningTests.java]: That's what I'm going for at the moment :)

**Comment by kasper-piskorski** [comment]
please retest this please - grakn-dist failure

**Pull request merged**